test/functional/ui/float_spec.lua

```diff
@@ -10809,6 +10809,86 @@ describe('float window', function()
+
+      if multigrid then
+        screen:expect{grid=[[
+        ## grid 1
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [3:----------------------------------------]|
+        ## grid 2
+          ^                                        |
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+        ## grid 3
+                                                  |
+
+        ## grid 4 (hidden)
+          {1:          }|
+          {2:~         }|
+        ]]}
+      else
+        screen:expect([[
+          ^                                        |
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+                                                  |
+        ]])
+      end
+      meths.win_set_config(win, {hide = false})
+      if multigrid then
+        screen:expect{grid=[[
+        ## grid 1
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [2:----------------------------------------]|
+          [3:----------------------------------------]|
+        ## grid 2
+          ^                                        |
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+          {0:~                                       }|
+        ## grid 3
+                                                  |
+
+        ## grid 4
+          {1:          }|
+          {2:~         }|
+        ]], float_pos = expected_pos}
+      else
+        screen:expect([[
+          ^                                        |
+          {0:~                                       }|
+          {0:~    }{1:          }{0:                         }|
+          {0:~    }{2:~         }{0:                         }|
+          {0:~                                       }|
+          {0:~                                       }|
+                                                  |
+        ]])
+      end
```

Comment:
I'd add `meths.win_set_config(win, {hide = true})` and add yet an expect, so show it can be hidden properly after already displayed.

Reply:
done.